civiform/civiform
# [deprecated] Create North Star "Ineligible" Page

### Description

Create North Star "Ineligible" Page
<img width="1021" alt="Ineligible" src="https://github.com/civiform/civiform/assets/23390490/ebc17cfa-4db6-4e65-858b-7ba7fefba36a">

## Release notes
n/a

### Checklist

#### General

Read the full guidelines for PRs [here](https://github.com/civiform/civiform/wiki/Technical-contribution-guide#creating-a-pull-request)


#### User visible changes

  - [x] Follow the workflow [here](<https://github.com/civiform/civiform/wiki/Internationalization-(i18n)#workflow-for-adding-or-editing-translations>)

### Instructions for manual testing

1. Log in as admin
2. Create a program
3. Add eligibility condition (e.g. a number question where the answer must be greater than 0)
4. Enable North Star UI: http://localhost:9000/dev/feature/NORTH_STAR_APPLICANT_UI/enable
5. Log in as client or TI
6. Apply to the program
7. Answer a question that fails the eligibility check (e.g. 0)
8. Click "continue"
9. Expect to view the "Ineligible" page

### Issue(s) this completes

#7486
